eva_sub_cli/exceptions/metadata_template_version_exception.py

```diff
@@ -0,0 +1,5 @@
+
+class MetadataTemplateVersionException(Exception):
```

Comment:
These don't need to be in individual files. You can keep them in one file called exceptions

Reply:
done